repondeur/zam_repondeur/templates/lectures_add.html

```diff
@@ -18,19 +19,15 @@ <h2>Ajouter une lecture...</h2>
 
     <form action="{{ request.path }}" method="POST">
         <div class="form-group">
-            <label for="dossier">Texte de loi</label>
-            <select id="select-dossier" name="dossier" class="form-control">
-                {% for dossier in dossiers %}
-                    <option value="{{ dossier.uid }}" {%- if loop.first %} selected{% endif %}>{{ dossier.titre }}</option>
-                {% endfor %}
+            <label for="dossier">Dossier législatif</label>
+            <select id="select-dossier" name="dossier" class="form-control" placeholder="Choisir un dossier législatif...">
+                <option value="" selected="selected"></option>
```

Comment:
Isn't it possible to keep the whole list here and flush it through JS on selectize load? It would make it accessible even if JS crashes somehow.